scene/gui/text_edit.cpp

```diff
@@ -2009,7 +2009,7 @@ void TextEdit::indent_selected_lines_right() {
 			// We don't really care where selection is - we just need to know indentation level at the beginning of the line.
 			int left = _find_first_non_whitespace_column_of_line(line_text);
 			int spaces_to_add = _calculate_spaces_till_next_right_indent(left);
-			// Since we will add this much spaces we want move whole selection and cursor by this much.
+			// Since we will add this much space we want to move whole selection and cursor by this much.
```

Comment:
`spaces` as a number of whitespace characters.

Reply:
changed it to this....
```
Since we will add these many spaces, we want to move the whole selection and cursor by this much.
```